components/form/introduction.rst

```diff
@@ -102,7 +102,7 @@ object to read data off of the correct PHP superglobals (i.e. ``$_POST`` or
     Now, when you process a form, you can pass the :class:`Symfony\\Component\\HttpFoundation\\Request`
     object to :method:`Symfony\\Component\\Form\\Form::handleRequest`::
 
-        $form->handleRequest($request);
+        $form->handleRequest();
```

Comment:
This shows the integration with the `HttpFoundation` component. It registers a `HttpFoundationRequestHandler` which needs a `Request` instance.